gateway/src/main.rs

```diff
@@ -38,18 +40,16 @@ async fn main() -> io::Result<()> {
         Sqlite::create_database(db_uri).await.unwrap();
     }
 
-    let docker_stats_path =
-        PathBuf::from_str(DOCKER_STATS_PATH).expect("to parse docker stats path");
-
-    // Return an error early if the docker stats path is not in the expected location.
-    if !docker_stats_path.exists() {
-        return Err(std::io::Error::new(
-            io::ErrorKind::NotFound,
-            format!(
-                "could not find docker stats at path: {:?}",
-                DOCKER_STATS_PATH
-            ),
-        ));
+    let mut docker_stats_source = DockerStatsSource::Bollard;
```

Comment:
I would probably add a log statement to know which method was chosen for the stats.